### PR TITLE
US3360 Registration Children Attribute Type tables and page

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

**Comment by silbermm** [comment]
we should find a way to limit the attributes in the dropdown list to only those values

**Comment by silbermm** [comment]
Should we limit the ability to add multiple of the same attribute?

File: CI/SQL/20160304110601_US3360-registration-children-attributes-attributes-table.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
+USE [MinistryPlatform]
+GO
+
+/****** Object:  Table [dbo].[cr_Registration_Children_Attributes]    Script Date: 3/4/2016 2:45:03 PM ******/
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
```
Comment (silbermm) [inline CI/SQL/20160304110601_US3360-registration-children-attributes-attributes-table.sql]: check for existence of table before creating

File: CI/SQL/20160304112101_US3360-registration-children-attributes-subpage.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
+USE [MinistryPlatform];
+
+SET IDENTITY_INSERT [dbo].[dp_Sub_Pages] ON;
+
```
Comment (silbermm) [inline CI/SQL/20160304112101_US3360-registration-children-attributes-subpage.sql]: Check for subpage id before creating

File: CI/SQL/20160304105301_US3360-registration-children-attributes.sql [hunk 3]
```diff
@@ -0,0 +1,38 @@
+USE [MinistryPlatform]
+GO
+
```
Comment (silbermm) [inline CI/SQL/20160304105301_US3360-registration-children-attributes.sql]: check for existence of all attributes before inserting

File: CI/SQL/20160304104501_US3360-registration-children-attribute-type.sql [hunk 4]
```diff
@@ -0,0 +1,17 @@
+USE [MinistryPlatform]
+GO
+
```
Comment (silbermm) [inline CI/SQL/20160304104501_US3360-registration-children-attribute-type.sql]: check for existence of attribute type before creating

File: CI/SQL/20160304104501_US3360-registration-children-attribute-type.sql [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+           ,[Domain_ID]
+           ,[Available_Online]
+           ,[__ExternalAttributeTypeID]
+           ,[Prevent_Multiple_Selection]
+           ,[Online_Sort_Order])
+     VALUES
+           ('GO Cincinnati - Registration Children'
+           ,0)
+GO
+
```
Comment (silbermm) [inline CI/SQL/20160304104501_US3360-registration-children-attribute-type.sql]: how do you know you'll get ID of 80 in the other environments?

File: CI/SQL/20160304112101_US3360-registration-children-attributes-subpage.sql (hunk 2 above)
Comment (IngageMaddoxT) [inline CI/SQL/20160304112101_US3360-registration-children-attributes-subpage.sql]: redundant with scripting model

**Pull request merged**